app/views/home/get_involved.html.erb

```diff
@@ -3,9 +3,9 @@
 
 <header class="block headings-block">
   <div class="inner-block floated-children">
-    <%= render partial: 'shared/heading',
-              locals: { big: true,
-                        heading: "Get involved" } %>
+    <%= render "govuk_publishing_components/components/title", {
+      title: "Get involved",
+    } %>
```

Comment:
👍🏼

Reply:
I'm not able to get this to work perfectly with the title component as it only supports a custom margin bottom that's fixed across views. I've decided to focus on replicating the mobile view over the desktop view to supplement this. I'll follow this methodology for the following spacing issues listed below.